src/java.base/unix/classes/sun/security/provider/NativePRNG.java

```diff
@@ -565,9 +665,18 @@ private void implNextBytes(byte[] data) {
                         }
                     data_len -= len;
                     }
+                    /*[ENDIF] CRIU_SUPPORT */
                 } catch (IOException e){
                     throw new ProviderException("nextBytes() failed", e);
                 }
         }
+
+        /*[IF CRIU_SUPPORT]*/
+        private void clearRNGState() {
+            if (mixRandom != null) {
+                mixRandom.clearState();
+            }
+        }
+        /*[ENDIF] CRIU_SUPPORT */
```

Comment:
The head stream has a blank line after this; there's no need for this file to be different.